guidelines/frontend-coding-standard/wrap-imported-library-into-custom-component.md

```diff
@@ -0,0 +1,42 @@
+# Encourage to wrap imported libraries into custom components
+
+It is better to wrap the imported external libraries into the custom elements.
+
+It is always implemented in elements such as button and textfield. For example,
+
+```js
+import MaterialChip from "@mui/material/Chip";
+
+export const CHIP_TYPE = {
+  filled: "filled",
+  outlined: "outlined",
+};
+
+export function Chip({
+  label,
+  icon,
+  deleteIcon,
+  variant = CHIP_TYPE.outlined,
+  onIconClick,
+}) {
+  return (
+    <MaterialChip
+      deleteIcon={deleteIcon}
+      icon={icon}
+      label={label}
+      sx={{ borderRadius: 2, height: "32px" }}
```

Comment:
this is also one of the advantage! We can introduce our own version of default behaviour of the library and hide the detail implementation behind the wrapper.